### ISSUE TITLE
Few issues with 8.2

### ISSUE THREAD

**Issue by jakeface1** [fafaraway/andromeda#25]
Installer gets stuck on 3/4 and can't progress, a reload causes the installer to restart. Target frame is not working.

**Comment by jakeface1** [comment]
Also when I /freeui unlock. Some of the UI elements are missing. Like raid and party.

**Comment by jabramek** [comment]
The only problem I am having issues with is missing frames. Only frames that appear are player and pet frames. 
```
1x [ADDON_ACTION_BLOCKED] AddOn 'FreeUI' tried to call the protected function 'FreeUI_ActionBar2:Show()'.
!BugGrabber\BugGrabber.lua:519: in function <!BugGrabber\BugGrabber.lua:519>
[C]: in function `Show'
FrameXML\SecureStateDriver.lua:100: in function <FrameXML\SecureStateDriver.lua:95>
FrameXML\SecureStateDriver.lua:127: in function <FrameXML\SecureStateDriver.lua:119>

Locals:
InCombatSkipped
```

**Comment by fafaraway** [comment]
Fixed with 583a7a6e41a9117b19f2089195d050d23e9abd01
